Lib/test/test_bytes.py

```diff
@@ -2060,6 +2060,24 @@ def __index__(self):
             self.assertEqual(instance.ba[0], ord("?"), "Assigned bytearray not altered")
             self.assertEqual(instance.new_ba, bytearray(0x180), "Wrong object altered")
 
+    def test_search_methods_reentrancy_raises_buffererror(self):
+        ba = bytearray(b"A")
+        class Evil:
+            def __index__(self):
+                ba.clear()
+                return 65  # ord('A')
+        with self.assertRaises(BufferError):
+            ba.find(Evil())
+        with self.assertRaises(BufferError):
+            ba.count(Evil())
+        with self.assertRaises(BufferError):
+            ba.index(Evil())
+        with self.assertRaises(BufferError):
+            ba.rindex(Evil())
+        with self.assertRaises(BufferError):
+            ba.rfind(Evil())
+
```

Comment:
According to the [PEP8](https://peps.python.org/pep-0008/#blank-lines), there should be only two blank lines.
```suggestion
```